core/src/main/clojure/xtdb/table_catalog_test.clj

```diff
@@ -29,21 +41,43 @@
         (xt/execute-tx node [[:put-docs :foo {:xt/id 2}]])
         (tu/finish-block! node)
 
-        (t/is (= [(os/->StoredObject "tables/public$foo/blocks/b00.binpb" 186)
-                  (os/->StoredObject "tables/public$foo/blocks/b01.binpb" 215)]
+        (t/is (= [(os/->StoredObject "tables/public$foo/blocks/b00.binpb" 286)
+                  (os/->StoredObject "tables/public$foo/blocks/b01.binpb" 415)]
                  (.listAllObjects bp (table-cat/->table-block-dir "public/foo"))))
 
-        (t/is (= [{:table-name "public/foo",
-                   :trie-key "l00-rc-b00",
-                   :data-file-size 1966}
-                  {:table-name "public/foo",
-                   :trie-key "l00-rc-b01",
-                   :data-file-size 1966}]
-                 (->> (.getByteArray bp (util/->path "tables/public$foo/blocks/b01.binpb"))
-                      TableBlock/parseFrom
-                      table-cat/<-table-block
-                      :current-tries
-                      (mapv added-trie->edn ))))
+        (let [current-tries (->> (.getByteArray bp (util/->path "tables/public$foo/blocks/b01.binpb"))
```

Comment:
(out of scope of this PR) these are the tests that I'd like us to move to Kotlin too, given the code it's testing is. I understand we don't yet have the same level of fixtures there that we do in Clojure (which is why I have also done the same as you here), so when we next come up for air let's figure out what we need to make this easier